app/src/main/java/zapsolutions/zap/fragments/SendBSDFragment.java

```diff
@@ -405,7 +405,7 @@ public void onClick(View v) {
                                     ZapLog.debug(LOG_TAG, throwable.getMessage());
 
                                     String errorPrefix = getResources().getString(R.string.error).toUpperCase() + ":";
-                                    String errormessage = throwable.getCause().getMessage().replace("UNKNOWN:", errorPrefix);
+                                    String errormessage = throwable.getMessage().replace("UNKNOWN:", errorPrefix);
```

Comment:
How does the error now look like?
If I recall correct I did it like this on purpose to shorten the error message.
In the end it was just:  Error:<Error Message>  that was displayed on screen.
What will now get displayed?